### DISCUSSION TITLE
Oqtane Version Upgrade without Migration

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#2569] [Q&A]
Hi,

We couldn’t migrate dotnet 6 to 7 since dotnet 7 is not LTS and also for company policy we can’t frequently migrate versions in the server. 

My current version of Oqtane is 3.3.1 and wanted to upgrade 4.0.0 (might need other newly added features) without migrating dotnet 7. How do I do that?

My suggestion, For every Oqtane’s migration why can’t we put in separate version without adding other features? So that, we can get only new features without migration. I don’t know whether it makes sense or not?

**Comment by sbwalker** [comment]
Please see #2465 as it has a longer conversation on this exact topic. Honestly I was very surprised that nobody spoke up with concerns about moving to .NET 7... as in my day job I know for a fact that none of our enterprise customers will be moving to .NET 7 because it is not LTS.
